### ISSUE TITLE
Enable administrator to configure IP addresses to which GlobaLeaks bound to

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/GlobaLeaks#1547]
This ticket is to enable the administrator from the web administration interface to configure IP addresses to which GlobaLeaks bound to.

This is a very common modification that's done by the sysadmin, having to modify /etc/default/globaleaks in multiple places, leading to configuration error.

The admin must be provided the availability to configure the IP address over which GlobaLeaks will listen, other than 127.0.0.1:
* during the configuration wizard
* after in the administration interface advanced settings

Actually, because there's no low-level, root-privileged supervisor, this change require a restart of the application. For that reason, after a change is applied, the administrator is given a permanent overlay message (that cannot be closed, unless the backend detect that the change has been applied), saying that the system administrator must issue as a linux root user the command "/etc/init.d/globaleaks restart" .

**Comment by evilaliv3** [comment]
As an update for this ticket currently we opted for having GlobaLeaks bind to 127.0.0.1 and 0.0.0.0 on differen ports depending on the need with the decision that all the configuration discussed in this ticket will have to be implemented by means of IP checks instead of edits to the bind calls.

Basically the idea is that in order to make possible change the configuration without requiring the application to restart, the application could instead just have socket binded but no service attached.

**Comment by fpietrosanti** [comment]
A different way to do that would be also to implement #1211

**Comment by evilaliv3** [comment]
I agree and my preference is to proceed for #1211 deprecating and closing this ticket .
